crates/nao/src/lib.rs

```diff
@@ -373,6 +373,7 @@ pub enum Network {
     SplD,
     SplE,
     SplF,
+    SplHulks,
```

Comment:
Please add this also to command_string in set_network.

Reply:
done